src/measure_theory/borel_space.lean

```diff
@@ -113,7 +113,8 @@ begin
       rw this, resetI,
       apply is_measurable.Union,
       exact λ _, (H _).compl _ } },
-  { simp, rintro _ a rfl,
+  { simp only [mem_range, exists_imp_distrib, forall_apply_eq_imp_iff'],
+    intro a,
     exact generate_measurable.basic _ is_open_Iio }
```

Comment:
I assume this is a reasonable fix, since I don't think I can rewrite this to put the simp last.

Reply:
Nevermind, I can eliminate this entirely